WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -25,17 +25,27 @@ class StoreOnboardingViewModel: ObservableObject {
     }
 
     var tasksForDisplay: [StoreOnboardingTaskViewModel] {
-        guard isRedacted == false else {
+        if isRedacted {
             return placeholderTasks
         }
-        guard !isExpanded else {
+
+        if isExpanded {
+            return taskViewModels
+        }
+
+        if !isExpanded && !shouldShowViewAllButton {
```

Comment:
nit: how about combining these two:
```suggestion
        if isExpanded || !shouldShowViewAllButton {
            return taskViewModels
        }
```

Reply:
Done in bb109d03a5e6ae12e60eeebaa315998031e13262